herbert_core/classes/MPIFramework/@ClusterSlurm/private/select_job_interactively_.m

```diff
@@ -0,0 +1,25 @@
+function job_id_str=select_job_interactively_(new_job_info)
+% function to select proper slurm job interactively
+% 
+% This should never happen
+fprintf(2,'*** WARNING: more then one new job have found in the slurm queue\n')
```

Comment:
its rather debug function placed there for benefit of developers and you may indeed see the output a) when debugging script step by step. b) When running 
test for it which is not a test but `extract_job_id_from_multistring_log_manually`

I indeed used this function when had bugs in the code. It may reappear if (e.g.) the format of slurm `squeue` command output changes.